extensions/amp-social-share/1.0/test/test-amp-social-share.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../amp-social-share';
+import {toggleExperiment} from '../../../../src/experiments';
+import {waitForChildPromise} from '../../../../src/dom';
+import {whenCalled} from '../../../../testing/test-helper.js';
+
+const BUTTON_SELECTOR = 'div[role="button"]';
+const WINDOW_FEATURES = 'resizable,scrollbars,width=640,height=480';
+
+describes.realWin(
+  'amp-social-share-v1.0',
+  {
+    amp: {
+      runtimeOn: true,
+      extensions: ['amp-social-share:1.0'],
+      canonicalUrl: 'https://canonicalexample.com/',
+    },
+  },
+  (env) => {
+    let win, doc;
+    let element;
+
+    const waitForRender = async () => {
```

Comment:
I *think* all our Bento `amp-` tests have a helper like this at this point, let's keep record of a follow up task to make a generic util.

Reply:
Agree!